src/search/engines/yahoo/params.go

```diff
@@ -5,16 +5,23 @@ import (
 )
 
 const (
-	paramKeyPage       = "b"
-	paramKeySafeSearch = "vm" // Can be "p" (disabled) or "r" (enabled).
+	// Variable params.
+	paramQueryK       = "p"
+	paramPageK        = "b"
+	cookieSafeSearchK = "vm" // Can be "p" (disabled) or "r" (enabled).
 
-	paramSafeSearchPrefix = "sB=v=1&pn=10&rw=new&userset=0"
+	// Constant params.
+	cookieSafeSearchPrefix = "sB=v=1&pn=10&rw=new&userset=0"
```

Comment:
Should `v=1` have `=` encoded?

Reply:
@k4lizen This is still an open topic, as I've left cookies intact but they should probably also get URL encoding treatment like URL params :)